### ISSUE TITLE
"Make a Copy" does not copy attached files

### ISSUE THREAD

**Issue by httpdev** [iodide-project/iodide#2741]
On alpha.iodide.io revision 1d87c1a1c8, when the user clicks on "Make a Copy", the notebook is cloned, but none of the files attached to it, which may prevent the notebook from working.

### What I Did

1. Create a new notebook
2. Go to Files > Add a file, say "test.png"
3. Verify that the file is accessible under /notebooks/<notebook_id>/files/test.png
4. Click on "Make a Copy"
5. With the new notebook, check if the file is accessible unter /notebooks/<Copy_of_notebook_id>/files/test.png

### What I Expected

The file should display.

### What Happened 

500 Internal Server Error message

### (if applicable) notebook that reproduces bug 

Original notebook: https://alpha.iodide.io/notebooks/4281/files/kaefer.png
Copy: https://alpha.iodide.io/notebooks/4283/files/kaefer.png

**Comment by wlach** [comment]
Yes this is a known issue, but I don't think we had an issue open for it. Thanks for filing!

**Comment by susan-shu-c** [comment]
Taking a quick look, seems to be able to add that [in here](https://github.com/iodide-project/iodide/blob/master/src/editor/actions/server-save-actions.js#L42)?

Possibly with the help [of this](https://github.com/iodide-project/iodide/blob/2cba3a5c9ec9768d32b834da60748aa46bccae71/src/shared/utils/file-operations.js#L71) or something in `/src/shared/utils/file-operations.js`
